.github/workflows/build.yml

```diff
@@ -70,7 +70,7 @@ jobs:
     - name: Update toolchain
       run: rustup update --no-self-update ${{ matrix.rust }} && rustup default ${{ matrix.rust }}
     - name: Run cargo check
-      run: cargo check -p windows
+      run: cargo check -p windows --features Foundation,Win32_Foundation,Win32_Graphics_Direct2D
```

Comment:
Unfortunately, we can't test `--all-features` as we do for `windows-sys` due to build server limitations (and a very greedy compiler) but at least this adds a bit more smoke testing for some common type system patterns that we can validate on the MSRV.